README-PLAYBOOK.md

```diff
@@ -1,164 +1,164 @@
 # VCert Playbook 
 
-VCert Playbook functionality solves the last mile problem for clients. Vcert does a great job fetching the certificate, 
-but clients often have to write scripts around it to install it in the right location and take action after the 
-certificate is downloaded. The Playbook functionality aims to support this use case natively. It makes pull provisioning 
-very easy for clients.
-
-## Key Features of Playbook.
-
-1. Users don't have to pass long command line arguments. They can now call VCert with playbook yaml files. A huge help 
-in automation and maintenance. 
-2. Users can specify, where the certificate goes once received from Venafi. It supports folder locations as well as 
-CAPI store. 
-3. Users can specify, any after installation actions need to be taken. E.g. Restart my apache or restart nginx or any 
-other script one would like to run after the certificate is available in the right location. 
-4. User can mention renewing before, this helps to run the script on daily basis as part of cronjob or automation, the 
-tool will check if certificate exists or the certificate requires renewal before fetching a new certificate based on 
-the values configured. 
-5. Supports many formats of certificate to be stored, PKCS12, JKS, PEM, and CAPI store.
-
-## Some use cases:
-1. A client having 100s of Microsoft servers can have this script as part of the scheduled job.  The tool will run, 
-check if the certificate  present in the CAPI store needs renewal, if yes, it will fetch a new certificate, and install 
-in CAPI. Tool can run any after install script as well. 
-2. Remote machines like ATMs, can use this tool to pull provisioning certificates on startup or regular intervals. Once 
-the certificate is received, it can run any post-processing scripts configured in the tool.
+VCert Playbook functionality solves the "last mile" problem. VCert has historically done a great job fetching certificates, but getting those certificates installed in the right location usually required custom scripting. The Playbook functionality addresses this use case natively. 
+
+> Throughout this article, we use _TLS Protect Datacenter_ to refer to Trust Protection Platform and _TLS Protect Cloud_ to refer to Venafi as as Service.
+
+## Key features of VCert playbook
+
+- You don't have to pass long command line arguments. You can now call VCert with playbook YAML files, which helps with automation and maintenance. 
```

Comment:
Here's a suggestion to make this more scannable, which customers love; also some minor edits for clarity:

- **Simplified Commands**: With VCert Playbook, you can avoid long command-line arguments. Instead, use playbook YAML files with VCert to enhance automation and maintenance ease.
  
- **Flexible Certificate Placement**: You can designate where to place the certificate and the format in which you want it once received from Venafi. VCert Playbook supports common keystore formats like PEM, JKS, PKCS#12, accommodating folder locations and the Windows CAPI store.
  
- **Post-Installation Actions**: Specify any actions that must be carried out after the certificate is installed. This includes restarting services like Apache or Nginx, or running any other scripts needed once the certificate is in the right location.
  
- **Smart Certificate Renewals**: VCert Playbook checks if a certificate already exists and whether it's due for renewal before requesting a new one. This functionality lets you run a script regularly without unnecessarily renewing certificates.
  
- **Compatibility**: VCert Playbook works seamlessly with both TLS Protect Cloud and TLS Protect Datacenter, ensuring it fits your particular environment.

Reply:
I should have made the bolded text sentence case, per our style guide